src/app/app/form/page.tsx

```diff
@@ -0,0 +1,13 @@
+import OnboardingForm from "@/components/OnboardingForm";
```

Comment:
I dont think is a good idea hava a URL like `/app/form`. Usually we have more than one form in an app, so I would prefer a route like `/app/onboarding`